lib/pageFetcher.ts

```diff
@@ -93,12 +93,18 @@ export class Fetcher {
         this.closed = true;
     }
 
-    async fetch<S>(node: Node, state: S, notifier: Notifier<FetchEvent, S>) {
+    async fetch<S>(node: Node, state: S, notifier: Notifier<FetchEvent, S>, includeMetadata: boolean) {
```

Comment:
why is this an option in the fetch function and not a parameter in the constructor?
If this option can be passed in the constructor it is also not needed anymore in the strategies

Reply:
Good point! Updated my code.